### PR TITLE
Project setup

### PR DESCRIPTION
**Pull request by znation2010**
Project requirements

-  You should follow these [design guidelines](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015), including:

        Colors.
        Typography: font face, size, and weight (we suggest using [Lato](https://www.latofonts.com/) as it is a free font similar to the one used in the original design).
        Layout: composition and space between elements.

-     The pages should look almost identical to the original design. Small adjustments like text or image changes are acceptable.

    You can use a CSS framework (for example Bootstrap) for styling, if you want to.

-     You should build only these 2 pages:

        The home page.
        The about page.

-     Each of these pages should have versions for 2 different screen sizes:

        Mobile: up to 768px wide.
        Desktop: 768px or wider.

-     Interactions

        Links
            The home page should have a link in the menu to the about page.
            The logo in the header links to the home page.
        Mobile menu
            When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
            There are no guidelines for the mobile menu in the docs, but you should implement it so it is consistent with the design (colors, typography, spacings, etc).
            The mobile menu should have a close (X) button that closes the menu.

-     Dynamic page

        The section "Featured speakers" should be created dynamically in JavaScript.
        You should use a JavaScript variable with the data about the speakers and use it when the page loads to create the HTML for this section dynamically.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        <a href="#">Logout</a>
+      </div>
+      <nav class="desktop" id="nav">
+        <div id="logo">
+          <a href="./index.html"><img src="./assets/NATO-Logo(4).png" alt="NATO logo"></a>
+        </div>
+        <div id="nav-links">
+          <ul>
+            <li>
+              <a id="nav-link-1" class="regular-link" href="./about.html">About</a>
```
Comment (joel-kalema) [inline about.html]: - [ ]  You did great work @znation2010   😃, however, in this `about.html` page we are not supposed to have again the `about` button in your navbar. What do you think if we change it to this `<a id="nav-link-1" class="regular-link" href="./index.html">Home</a>` to facilitate the user to go back to the `home` page 👍. Kindly fix it and make sure it is the same for the `mobile menu` 💪.  


- [ ] Also your buttons for `about.html` are taking the same color as a hover button. Kindly fix it 👍 

Screenshot 2023-05-31 121439


- [ ] Kindly, make sure, when we are at the desktop version the nav suppose to display none automatically 👍 . let's make it responsive 💪 

Screenshot 2023-05-31 121936

**Review by joel-kalema — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @znation2010 
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Your design looks good :100: :heavy_check_mark: 
- Good PR description :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 
- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @joel-kalema  when you comment so I can receive the notification or use slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: about.html (hunk 1 above)
Comment (znation2010) [reply about.html]: hi brother can you tell what to do to make  the desktop version the nav suppose to display none automatically 👍 . let's make it responsive

Comment (joel-kalema) [reply about.html]: Hi @znation2010, as you worked on responsiveness, you may just detect the screen and set a value to display the nav 💪

**Review by tufoinnkuo10 — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @znation2010 

Your project is complete! Great job making the changes requested🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍
- Descriptive Pull Request ✔️✔️
- well deployed ✔️ 
- Great Readme file✔️✔️
- Wonderful design✔️✔️
- well Responsive✔️✔️
- No linter errors✔️✔️

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 ✔️✔️

tufoin

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**